FirebaseStorage/Sources/Storage.swift

```diff
@@ -340,3 +325,30 @@ import FirebaseCore
     }
   }
 }
+
+extension Storage {
```

Comment:
Why an extension?

Reply:
I moved it into the original scope. It was a stylistic choice. When it comes to nest classes that have a direct couple relationship with the original class, it's probably better to keep it in the original scope than in an extension. Thanks for questioning.